src/zero-md.js

```diff
@@ -165,12 +165,11 @@ class ZeroMd extends HTMLElement {
   _hijackLinks(ev) {
     let path = ev.path || ev.composedPath();
     if (path[0].tagName !== 'A') { return; }
-    ev.preventDefault();
-    if (ev.metaKey) {
-      window.open(window.location.href  + path[0].hash, '_blank');
-    } else {
+
+    const link = path[0];
+    if (link.hash && link.href === window.location.href) {
       this._scrollTo(path[0].hash);
-      window.location = path[0].href;
+      ev.preventDefault();
     }
   }
```

Comment:
Thanks for the quick work! It suddenly occurred to be that we should still check for CMD+click for anchor-link-scroll cases. The expected result if a user CMD+clicks on a hash link is that a new tab opens with the page duplicated and scrolled to the correct heading. 

Agree that we don't have to `preventDefault` none hash-link cases. 

Another thought is the check `link.hash && link.href === window.location.href` might not correctly isolate out valid hash-link cases. Take for example if a user's browser is at "https://example.com/page", and the link href is "https://example.com/page#section1"...

Perhaps this function can be refactored to:

```js
  _hijackLinks(ev) {
    let path = ev.path || ev.composedPath();
    if (path[0].tagName !== 'A') { return; }
    // check that it's a hash-link case
    if (path[0].hash && (path[0].origin + path[0].pathname === window.location.origin + window.location.pathname)) {
      ev.preventDefault();
      if (ev.metaKey) {
        window.open(window.location.href  + path[0].hash, '_blank');
      } else
        this._scrollTo(path[0].hash);
        window.location = path[0].href;
      }
    }
  }
```

Reply:
Ok good point, I will refactor